package.json

```diff
@@ -14,7 +14,8 @@
   },
   "scripts": {
     "test": "xo && mocha",
-    "bench": "matcha bench.js"
+    "bench": "matcha bench.js",
+    "jshint": "jshint index.js"
```

Comment:
This project already uses [XO](https://github.com/sindresorhus/xo) linter. https://github.com/sindresorhus/object-assign/blob/b0c40d37cbc43e89ad3326a9bad4c6b3133ba6d3/package.json#L16